packages/ui/src/components/label-marker.tsx

```diff
@@ -25,23 +29,73 @@ export const LabelMarkerColor = {
   [ColorsEnum.MINT]: 'bg-label-background-mint text-label-foreground-mint',
   [ColorsEnum.LIME]: 'bg-label-background-lime text-label-foreground-lime'
 }
+const deleteButtonColor = {
+  [ColorsEnum.RED]: 'text-label-foreground-red hover:opacity-60',
+  [ColorsEnum.BLUE]: 'text-label-foreground-blue hover:opacity-60',
+  [ColorsEnum.GREEN]: 'text-label-foreground-green hover:opacity-60',
+  [ColorsEnum.YELLOW]: 'text-label-foreground-yellow hover:opacity-60',
+  [ColorsEnum.PURPLE]: 'text-label-foreground-purple hover:opacity-60',
+  [ColorsEnum.PINK]: 'text-label-foreground-pink hover:opacity-60',
+  [ColorsEnum.VIOLET]: 'text-label-foreground-violet hover:opacity-60',
+  [ColorsEnum.INDIGO]: 'text-label-foreground-indigo hover:opacity-60',
+  [ColorsEnum.CYAN]: 'text-label-foreground-cyan hover:opacity-60',
+  [ColorsEnum.ORANGE]: 'text-label-foreground-orange hover:opacity-60',
+  [ColorsEnum.BROWN]: 'text-label-foreground-brown hover:opacity-60',
+  [ColorsEnum.MINT]: 'text-label-foreground-mint hover:opacity-60',
+  [ColorsEnum.LIME]: 'text-label-foreground-lime hover:opacity-60'
+}
+
+export const LabelMarker: FC<LabelMarkerProps> = ({
+  type = LabelType.STATIC,
+  label,
+  value,
+  color,
+  counter,
+  onDelete,
+  className
+}) => {
+  const isDynamic = type === LabelType.DYNAMIC
+  const isWithExtraContent = !!value || !!counter
+  const isWithDeleteButton = !!onDelete
```

Comment:
You can totally ignore this since it makes zero difference... instead of `isWith`, `has` reads better IMO
```suggestion
  const hasExtraContent = !!value || !!counter
  const hasDeleteButton = !!onDelete
```